### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by rainyday-var**
in case ini file has lines like this, incoming value will be as int and isdigit raises an error.

try it out

[some]
key = '10'
key1 = 10

### CODE REVIEW

File: ini/__init__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 
 def _parse_value(value):
-    if value.isdigit() or (value.startswith('-') and value[1:].isdigit()):
+    if str(value).isdigit() or (value.startswith('-') and value[1:].isdigit()):
```
Comment (vrumger) [inline ini/__init__.py:8]:
```suggestion
    if isinstance(value, int) or value.isdigit() or (value.startswith('-') and value[1:].isdigit()):
```

**Review by vrumger — CHANGES_REQUESTED** [review]

**Comment by vrumger** [comment]
good catch - thanks!

**Comment by dziaineka** [comment]
@vrumger would you be so kind to update pypi package https://pypi.org/project/ini-parser/?

**Comment by vrumger** [comment]
@dziaineka yeah, I'm pretty busy this weekend. I'll update it Sunday morning :)

**Comment by vrumger** [comment]
@dziaineka I've updated it on pypi :)

**Pull request merged**